src/components/intent.ts

```diff
@@ -375,9 +376,18 @@ export class Intent {
         // eslint-disable-next-line camelcase
         : Promise<{event_id: string}> {
         if (this.encryption) {
-            // We *need* to sync before we can send a message.
-            await this.ensureRegistered();
-            await this.encryption.ensureClientSyncingCallback();
+            let encrypted = false;
```

Comment:
We previously started syncing any client that sent a message to a room, but that's inefficent if the room is unencrypted as we do not need to sync.

This code checks to see if a room is encrypted.